lib/dpl/provider/elastic_beanstalk.rb

```diff
@@ -71,7 +71,10 @@ def version_label
       end
 
       def version_description
-        context.env['ELASTIC_BEANSTALK_DESCRIPTION'] || commit_msg
+        description = context.env['ELASTIC_BEANSTALK_DESCRIPTION'] || commit_msg
+        description.gsub!(/[^\u0009\u000A\u000D\u0020-\uD7FF\uE000-\uFFFD\u10000-\u10FFFF]/, "") and
```

Comment:
This character class is too permissive. Some (most?) reported characters fall in this range (`\u{10000}-\u{10FFFF}`) and will be passed through, and cause the exception.

I've opened https://github.com/aws/aws-sdk-ruby/issues/1502 to get a better answer.